sass/_clients.scss

```diff
@@ -193,6 +202,10 @@ body:has(.clients-card-deck > div > .client-checkbox:checked) {
                     user-select: none;
                 }
             }
+
+            &.display {
+                display: block;
+            }
```

Comment:
Instead of a default `display: none` and override, I personally prefer `&:not(.display)` style rules. In this case you wouldn't have to set the `display` property for the visible case then, because `block` is default (for most things).

Reply:
Fixed in 3d8a1632